app/src/main-process/menu/build-default-menu.ts

```diff
@@ -311,6 +311,15 @@ export function buildDefaultMenu(
     },
   }
 
+  const contactSupportItem: Electron.MenuItemConstructorOptions = {
+    label: __DARWIN__ ? 'Contact GitHub Support…' : 'Contact GitHub support…',
```

Comment:
Can we add an access key in the windows label please? Perhaps `&Contact GitHub support` as that doesn't seem to be used in the help menu?